package.json

```diff
@@ -43,6 +43,7 @@
         "@swimlane/ngx-charts": "20.3.0",
         "@swimlane/ngx-graph": "8.0.3",
         "ace-builds": "1.22.0",
+        "angular2-draggable": "^16.0.0",
```

Comment:
Make sure to discuss new dependencies with Stephan and/or Patrick.

Reply:
Patrick said that it's fine